pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>98</version>
+        <version>99</version>
```

Comment:
The latest airbase is 98. https://github.com/airlift/airbase/releases

Reply:
Thanks for pointing out, in the changes 99 is already mentioned, so I was under the impression that it was already released.